html/index.html

```diff
@@ -0,0 +1,44 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="../styles/main.css">
+    <title>WebStore</title>
+</head>
+<body>
+<div class="page-wrapper">
+    <header id="header">
+        <div>
+            <a href="index.html" id="logo"><img src="../img/logo/logo.png" alt="logo"></a>
+        </div>
+        <div id="menuToggle">
+            МЕНЮ
+        </div>
+        <nav id="menu" class="menu1">
+            <ul class="menu">
+                <li><a href="catalog.html">Women</a></li>
+                <li><a href="catalog.html">Men</a></li>
+                <li><a href="catalog.html">Handbags</a></li>
+                <li><a href="catalog.html">Accessories</a></li>
+                <li><a href="catalog.html">Sale</a></li>
+                <li><a href="catalog.html">New Arrivals</a></li>
+                <li><a href="catalog.html">Clearance</a></li>
+                <li><a href="catalog.html">Store Locator</a></li>
+            </ul>
+        </nav>
+        <div class="contacts">
+            <ul class="contactsList">
```

Comment:
Define, for yourself, single and constant rule for naming selectors. If you use camel case, you should use it everywhere. I advice you to use **underscore**, e.g. contacts_list, header_sign-up.